python/tvm/autotvm/task/relay_integration.py

```diff
@@ -153,15 +154,25 @@ def extract_from_multiple_program(funcs, params, ops, target, target_host=None,
 
         logger.disabled = old_state
 
+    # convert *topi op to template key* map to *task name to template key* map
+    task_keys = dict()
```

Comment:
Is there any particular concern to use `dict()` instead of `{}` to create a dictionary? `{}` would be faster in general [[ref](https://stackoverflow.com/questions/5790860/and-vs-list-and-dict-which-is-better)]. Also, I think the naming is a bit confusing. I would prefer a longer but clearer one such as `task_name_to_keys` or something like that.

Reply:
> Is there any particular concern to use dict() instead of {} to create a dictionary?
I think `dict()` is more easy to read... C++ guys...

I agree with your comments generally, will update.